script/deploy

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+## Validations
+[ -z "${DOCKER_USER}" ] && { echo "Need to set DOCKER_USER"; exit 0; }
```

Comment:
I could see an argument for the validations to `exit 1`, but not for the branch check. Split it down the center?